<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1293</width>
    <height>854</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1291</width>
      <height>801</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QPushButton" name="Reload">
       <property name="text">
        <string>Reload</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="UpdateAll">
       <property name="text">
        <string>Update All</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="7">
      <widget class="QTableWidget" name="Table">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="cursor" stdset="0">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <row>
        <property name="text">
         <string>71AW01</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>71AW02</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>71AW03</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>71AW04</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>71AW05</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>71WS01</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>71WS02</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>71WS03</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>71WS04</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>71WS05</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>71WS06</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>TCLI01</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>TCLI02</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>TCLI03</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>TCLI04</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>TCLI05</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>TCLI06</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>71ES02</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>71ES03</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>VIRT03</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>HIST1A</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>NAS_SRV</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>EPOSRV</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>WSUS</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>NETMON</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>PI_MAG</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>PIMAG_IN</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>SYSADV</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>AUTOSAVE</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>PRIMDC</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>SECNDC</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string>Machine</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>OS</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>KB</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Servicing Stack</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Cumulative</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Install Status</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Notes</string>
        </property>
       </column>
       <item row="0" column="0">
        <property name="text">
         <string>71AW01</string>
        </property>
       </item>
       <item row="1" column="0">
        <property name="text">
         <string>71AW02</string>
        </property>
       </item>
       <item row="2" column="0">
        <property name="text">
         <string>71AW03</string>
        </property>
       </item>
       <item row="3" column="0">
        <property name="text">
         <string>71AW04</string>
        </property>
       </item>
       <item row="4" column="0">
        <property name="text">
         <string>71AW05</string>
        </property>
       </item>
       <item row="5" column="0">
        <property name="text">
         <string>71WS01</string>
        </property>
       </item>
       <item row="6" column="0">
        <property name="text">
         <string>71WS02</string>
        </property>
       </item>
       <item row="7" column="0">
        <property name="text">
         <string>71WS03</string>
        </property>
       </item>
       <item row="8" column="0">
        <property name="text">
         <string>71WS04</string>
        </property>
       </item>
       <item row="9" column="0">
        <property name="text">
         <string>71WS05</string>
        </property>
       </item>
       <item row="10" column="0">
        <property name="text">
         <string>71WS06</string>
        </property>
       </item>
       <item row="11" column="0">
        <property name="text">
         <string>TCLI01</string>
        </property>
       </item>
       <item row="12" column="0">
        <property name="text">
         <string>TCLI02</string>
        </property>
       </item>
       <item row="13" column="0">
        <property name="text">
         <string>TCLI03</string>
        </property>
       </item>
       <item row="14" column="0">
        <property name="text">
         <string>TCLI04</string>
        </property>
       </item>
       <item row="15" column="0">
        <property name="text">
         <string>TCLI05</string>
        </property>
       </item>
       <item row="16" column="0">
        <property name="text">
         <string>TCLI06</string>
        </property>
       </item>
       <item row="17" column="0">
        <property name="text">
         <string>71ES02</string>
        </property>
       </item>
       <item row="18" column="0">
        <property name="text">
         <string>71ES03</string>
        </property>
       </item>
       <item row="19" column="0">
        <property name="text">
         <string>VIRT03</string>
        </property>
       </item>
       <item row="20" column="0">
        <property name="text">
         <string>HIST1A</string>
        </property>
       </item>
       <item row="21" column="0">
        <property name="text">
         <string>NAS_SRV</string>
        </property>
       </item>
       <item row="22" column="0">
        <property name="text">
         <string>EPOSRV</string>
        </property>
       </item>
       <item row="23" column="0">
        <property name="text">
         <string>WSUS</string>
        </property>
       </item>
       <item row="24" column="0">
        <property name="text">
         <string>NETMON</string>
        </property>
       </item>
       <item row="25" column="0">
        <property name="text">
         <string>PI_MAG</string>
        </property>
       </item>
       <item row="26" column="0">
        <property name="text">
         <string>PIMAG_IN</string>
        </property>
       </item>
       <item row="27" column="0">
        <property name="text">
         <string>SYSADV</string>
        </property>
       </item>
       <item row="28" column="0">
        <property name="text">
         <string>AUTOSAVE</string>
        </property>
       </item>
       <item row="29" column="0">
        <property name="text">
         <string>PRIMDC</string>
        </property>
       </item>
       <item row="30" column="0">
        <property name="text">
         <string>SECNDC</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QPushButton" name="Quit">
       <property name="text">
        <string>Quit</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="UpdateSelected">
       <property name="text">
        <string>Update Selected</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="Test">
       <property name="text">
        <string>Schedule The Updates</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QPushButton" name="ClearSelections">
       <property name="text">
        <string>Clear Selections</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1293</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEdit_Table">
    <property name="title">
     <string>Edit Table</string>
    </property>
    <addaction name="actionEdit_Table"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuEdit_Table"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionEdit_Table">
   <property name="text">
    <string>Edit Table</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
